python/ctsm/site_and_regional/tower_site.py

```diff
@@ -257,11 +257,11 @@ def run_case(
         user_version,
         tower_type,
         user_mods_dirs,
-        overwrite=False,
-        setup_only=False,
-        no_batch=False,
-        rerun=False,
-        experiment=False,
+        overwrite,
```

Comment:
So this is changing the defaults that allow these options to NOT to be set, to be required to be set. I suppose if these options are always provided there's no reason to have the default. But, I wasn't sure why this change was being made?

Reply:
These values are not always set by the user. I was noticing that tower_site was not correctly pulling information from neon_site if neon_site didn't set all of the parameters. If there's a better way to do this, I'm all ears :)

Reply:
Instead of the default parameter being set in neon_site, it's now in tower_site.

Reply:
Instead of the default parameter being set in neon_site, it's now in tower_site.